documents-d-archeologie-francaise.csl

```diff
@@ -424,214 +406,164 @@
   </citation>
   <bibliography>
     <sort>
-      <key macro="responsability"/>
+      <key macro="author-short"/>
       <key macro="year-date"/>
       <key macro="title"/>
     </sort>
     <layout>
-      <group prefix="" suffix="" delimiter=" ">
+      <group prefix="" suffix="&#9;" delimiter=" ">
```

Comment:
Indeed, last Zotero version and MS Word 15

Le 19/01/2016 18:37, Rintze M. Zelle a écrit :
>
> In documents-d-archeologie-francaise.csl 
> <https://github.com/citation-style-language/styles/pull/1866#discussion_r50145590>:
>
> >        <key macro="year-date"/>
> >        <key macro="title"/>
> >      </sort>
> >      <layout>
> > -      <group prefix="" suffix="" delimiter=" ">
> > +      <group prefix="" suffix="&#9;" delimiter=" ">
>
> And this is with Zotero and MS Word, right?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/citation-style-language/styles/pull/1866/files#r50145590>.
>